eq-author/src/App/index.js

```diff
@@ -20,6 +20,10 @@ import QuestionnaireDesignPage from "./QuestionnaireDesignPage";
 import NotFoundPage from "./NotFoundPage";
 import ErrorBoundary from "./ErrorBoundary";
 
+import * as Sentry from "@sentry/browser";
```

Comment:
You can move this config to `src/index` which is where the rest of the set up is. You can also import `src/config` which allows for the environment variable to be overridden in dev.